app/scripts/controllers/metametrics-data-deletion/metametrics-data-deletion.ts

```diff
@@ -0,0 +1,125 @@
+import { ObservableStore } from '@metamask/obs-store';
+import MetaMetricsController from '../metametrics';
+import {
+  createDataDeletionRegulationTask,
+  fetchDeletionRegulationStatus,
+} from './services/services';
+
+export type DataDeleteDate = string;
+export type DataDeleteRegulationId = string;
+
+export type RegulationId = Record<string, string>;
+export type CurrentRegulationStatus = Record<string, Record<string, string>>;
+export type DeleteRegulationAPIResponse = {
+  data: RegulationId | CurrentRegulationStatus;
+};
+
+export enum DeleteRegulationStatus {
```

Comment:
Are there API docs somewhere that can explain what status exist, and what they mean?

Reply:
The existing statuses are mentioned here:

https://docs.segmentapis.com/tag/Deletion-and-Suppression#operation/listRegulationsFromSource

Reply:
Could you update the comment to reference these docs?